skin.json

```diff
@@ -57,7 +61,12 @@
 				"assets/scripts/foundation/foundation.alert.js",
 				"assets/scripts/foundation/foundation.accordion.js",
 				"assets/scripts/foundation/foundation.abide.js",
-				"foreground/assets/scripts/foreground.js"
```

Comment:
I went ahead and removed those three since they are unused by this repo and cause extra server load. I am fiding a tiny bit speed bump removing them.